src/index.js

```diff
@@ -8,52 +10,29 @@ const fs = require("fs");
  * @param {*} mode the mode to use (prefix or suffix)
  */
 function setEnvWithValue(key, contextOrBranch, mode) {
-  let foundOne = false;
-  let found;
-
-  if (mode === "prefix") {
-    const prefixedEnvVar = `${contextOrBranch}_${key}`;
-
-    if (process.env[prefixedEnvVar]) {
-      console.log(`Setting ${key} to the value from ${prefixedEnvVar}.`);
-      process.env[key] = process.env[prefixedEnvVar];
-      found = `${key}=${process.env[prefixedEnvVar]}`;
-    }
-  } else {
-    const suffixedEnvVar = `${key}_${contextOrBranch}`;
+  const envVar = mode === 'prefix' ? `${contextOrBranch}_${key}` : `${key}_${contextOrBranch}`;
 
-    if (process.env[suffixedEnvVar]) {
-      console.log(`Setting ${key} to the value from ${suffixedEnvVar}.`);
-      process.env[key] = process.env[suffixedEnvVar];
-      found = `${key}=${process.env[suffixedEnvVar]}`;
-    }
+  if (!process.env[envVar]) {
+    return ''
```

Comment:
What do you think about returning `undefined` here to avoid the mental leap of casting `""` to `false`?

```suggestion
    return;
```